tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "esModuleInterop": true,
     "checkJs": false,
     "downlevelIteration": true,
-    "lib": ["esnext", "dom"],
+    "lib": ["esnext", "dom", "dom.iterable"],
```

Comment:
This was required to support `formData.entries()`, used when we serialize form data (`Object.fromEntries(formData)`)